pom.xml

```diff
@@ -51,7 +51,7 @@
         <jenkins-test-harness.version>1.625.3</jenkins-test-harness.version>
         <release.skipTests>false</release.skipTests>
         <maven.javadoc.skip>true</maven.javadoc.skip>
-        <findbugs-maven-plugin.version>3.0.2</findbugs-maven-plugin.version>
+        <findbugs-maven-plugin.version>3.0.5</findbugs-maven-plugin.version>
```

Comment:
Should we do this in a separate PR?
Or is it necessary for the fix?

Reply:
It is not necessary for this fix. I added it because I don't think it hurts, but it can be moved to a separate PR, or even not changed if there is any reason to keep it to 3.0.2.

Reply:
Just about the atomicity of the commits.